shaikram_abdulguro/activity1/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css">
+    <title>Log In</title>
+</head>
+<style>
+    body{
+        background: #2d2f30;
+    }
+    h1{
+        color: blue;
+    }
+    h3{
+        color: gray;
+    }
+    form{
+        margin-top: 5em;
+        width:500px;
+        height:auto;
+        border: 1px solid grey;
+        background:#fff;
+        padding: 2em;
+        border-radius: 3px;
+    }
+    h3, span, label{
+        float: left;
+    }
+</style>
+<body>
+    <center>
```

Comment:
center tag is deprecated already,
Bootstrap has already built in center which is align-self-center, text-center, justify-content-center